packages/create-contentful-app/template/src/components/EntryEditor.spec.tsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import EntryEditor from './EntryEditor';
+import { render } from '@testing-library/react';
+
+describe('Entry component', () => {
+  it('Matches snapshot', () => {
+    const component = render(<EntryEditor />);
+
+    expect(component.container).toMatchSnapshot();
```

Comment:
Does this test provide any value? I don't think it is necessary

Reply:
I think the benefit of these is that it provides spec files for the user, and shows them the basic syntax of writing a test.